seb_openedx/tests/test_utils.py

```diff
@@ -0,0 +1,20 @@
+""" Testing utils """
+# pylint: disable=attribute-defined-outside-init, invalid-name
+
+
+def get_courseware_module():
+    """ helper for settings.test fake backend """
+    class Object(object):
+        """ dummy class """
+        pass
+
+    def get_course(course_id, depth=0, **kwargs):
+        """ dummy get_course function """
+        return None
+    fake_courseware_module = Object()
```

Comment:
Would a MagicMock not do all this by default?

Reply:
then you only need to tell it to patch `fake_courseware_module.courses.get_course = get_course`

Reply:
Ok